jenkins/Jenkinsfile

```diff
@@ -260,14 +258,17 @@ pipeline {
                                 if (GENERATE_DATAPROC_IMAGE == "true" && dataprocImages.contains(image)) {
                                     println("Using version $version for $image in this build of custom Dataproc images")
                                     sh """
-                                        sed -i "s/$image.*/$image:$version\\"/" jenkins/dataproc-custom-images/prepare-custom-leonardo-jupyter-dataproc-image.sh
+                                       if [ $image != 'terra-jupyter-aou' ]; then
+                                          sed -i "s/$image.*/$image:$version\\"/" jenkins/dataproc-custom-images/prepare-custom-leonardo-jupyter-dataproc-image.sh
+                                       fi
                                     """
                                 }
 
                                 if (GENERATE_GCE_IMAGE == "true" && gceImages.contains(image)) {
                                     println("Using version $version for $image in this build of custom GCE images")
+                                    // Not replace aou version because we want to fix bake 2 aou versions until aou migrate to new version
                                     sh """
-                                        sed -i "s/$image.*/$image:$version\\"/" jenkins/gce-custom-images/prepare_custom_leonardo_gce_image.sh
+                                       sed -i "s/$image.*/$image:$version\\"/" jenkins/gce-custom-images/prepare_custom_leonardo_gce_image.sh
```

Comment:
you will need the if for AOU here as well

Reply:
aou is still using dataproc..they'll probably migrate to newer aou image before migrating to gce..so I just made gce to cache latest aou image....